packages/test/local-server-stress-tests/src/test/localServerStress.spec.ts

```diff
@@ -12,24 +12,26 @@ import {
 	LocalServerStressModel,
 } from "../localServerStressHarness";
 
-describe("Local Server Stress", () => {
-	const model: LocalServerStressModel<StressOperations> = {
-		workloadName: "default",
-		generatorFactory: () => takeAsync(100, makeGenerator()),
-		reducer,
-		validateConsistency: validateConsistencyOfAllDDS,
-	};
+for (let i = 0; i < 150; i++) {
```

Comment:
Keeping the loop was intentional? Feels like it would make for weird (repeated suite/test id) test reports.

Reply:
no. my push failed. fixed!